tests/utils/cms/endpoints.go

```diff
@@ -0,0 +1,174 @@
+package cms
+
+// Cluster Management Service
+const (
+	// metadata
+	clusterMgmtURL        = "/api/clusters_mgmt"
+	clusterMgmtVersionURL = "/api/clusters_mgmt/v1"
+	clusterOpenapiURL     = "/api/clusters_mgmt/v1/openapi"
+
+	// Clusters
+	clusterURL            = "/api/clusters_mgmt/v1/clusters"
```

Comment:
is it used anywhere? I think the sdk abstracts away the endpoints so the tests don't really need to know those

Reply:
@osherdp I added the bunch of API endpoints for the next tests as we are going to verify terraform apply results by API responses

Reply:
@osherdp removed unused endpoints - will add if needed

Reply:
iiuc you want to check the state in the server to make sure terraform did the right change
in this case, you don't really need to hardcode any API endpoint. SDK allows querying the server and from what I can see you already use it exclusively in ``cms.go``

Reply:
@osherdp Sorry, I don't understand how to implement it?
I have func RetrieveMachineTypeDetail(connection *client.Connection, machineTypeID string) (*client.Response, error) {
	return connection.Get().Path(fmt.Sprintf(machineTypeIDURL, machineTypeID)).Send()
}
and 	machineTypeIDURL = "/api/clusters_mgmt/v1/machine_types/%s"
endpoint for this. How can it be queried?

Reply:
an example for a similar concept:
https://github.com/openshift/rosa/blob/c726dd090964aaa737dfd22575cba25e5c1ffa51/pkg/ocm/machines.go#L64-L75

something like
```golang
connection.ClustersMgmt().V1().MachineTypes().List()...
```
can yield all machine types

or if you want a specific one by id:
```golang
connection.ClustersMgmt().V1().MachineTypes().MachineType("id")...
```